app/views/devise/registrations/edit.html.haml

```diff
@@ -66,6 +66,22 @@
       %p.mt-1.p Danger zone! Once you delete you account, there is no going back.
 
     .card.md:col-span-2
-      .card-body
+      .card-body{"x-data" => "{ button: true, open: false }"}
```

Comment:
why do we need button and open? is it not just one action and then it opens? 

how is this handed where the password the user entered is wrong?

Reply:
> why do we need button and open? is it not just one action and then it opens?
> 
> how is this handed where the password the user entered is wrong?

On clicking the button 'Delete Feedback', we need to hide the same button before we can show the other text. That's why we have two different components in Alpine